README.md

```diff
@@ -156,25 +156,7 @@ Works with `sonar-scanner` and `gradle`
 
 .gitlab-ci.yml sample for Maven project, comment last commit:
 
-```yml
-sonarqube_master_job:
-  stage: test
-  only:
-    - master
-  script:
-    - mvn --batch-mode verify sonar:sonar -Dsonar.host.url=$SONAR_URL -Dsonar.login=$SONAR_LOGIN
-
-sonarqube_preview_feature_job:
-  stage: test
-  only:
-    - /^feature\/*/
-  script:
-    - git checkout origin/master
-    - git merge $CI_COMMIT_SHA --no-commit --no-ff
-    - mvn --batch-mode verify sonar:sonar -Dsonar.host.url=$SONAR_URL -Dsonar.login=$SONAR_LOGIN -Dsonar.analysis.mode=preview -Dsonar.gitlab.project_id=$CI_PROJECT_PATH -Dsonar.gitlab.commit_sha=$CI_COMMIT_SHA -Dsonar.gitlab.ref_name=$CI_COMMIT_REF_NAME
-```
-
-With quality gate
+This plugin requires the [Community Branch Plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin)  for SonarQube CE
```

Comment:
Based on [sonarqube-community-branch-plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin) we should have to configure the `sonar.branch.name` parameter to enable PR decoration. We should add those examples?

Reply:
I see..

I don't know if this is necessary.. Thanks!